app/controllers/concerns/two_factor_authenticatable_methods.rb

```diff
@@ -12,16 +12,19 @@ def auth_methods_session
   end
 
   def handle_verification_for_authentication_context(result:, auth_method:, extra_analytics: nil)
+    increment_mfa_selection_attempt_count(auth_method)
     analytics.multi_factor_auth(
       **result.to_h,
       multi_factor_auth_method: auth_method,
       enabled_mfa_methods_count: mfa_context.enabled_mfa_methods_count,
       new_device: new_device?,
       **extra_analytics.to_h,
+      attempts: user_session[:mfa_attempts][:attempts],
```

Comment:
Maybe we want to use the `mfa_attempts_count` helper here as well?

```suggestion
      attempts: mfa_attempts_count,
```